pygam/pygam.py

```diff
@@ -1305,23 +1319,24 @@ def _pirls(self, X, y, weights):
         # initialize GLM coefficients if model is not yet fitted
         if (not self._is_fitted or
             len(self.coef_) != sum(self._n_coeffs) or
-            not np.isfinite(self.coef_).all()):
+                not np.isfinite(self.coef_).all()):
 
-           # initialize the model
-           self.coef_ = self._initial_estimate(X, y)
+            # initialize the model
```

Comment:
@maorn good catch.
yes, in general these formatting changes are helpful and appear to make the code more legible.